config/bidderinfo.go

```diff
@@ -52,6 +52,15 @@ type BidderInfo struct {
 	OpenRTB             *OpenRTBInfo `yaml:"openrtb" mapstructure:"openrtb"`
 }
 
+type bidderInfoNullableFields struct {
```

Comment:
Nitpick: In Go, this should be `NillableFields`

Reply:
https://github.com/prebid/prebid-server/pull/3000/commits/c6be49c2ffcda9511cbe40163240a27aeec1bf52